src/utilities/filepatching.rs

```diff
@@ -13,21 +13,22 @@ impl FilePatching {
         let mut dir = dir;
         dir.push(&p.mainprefix);
         let mut target = dir.clone();
-        target.push(&p.modname()?);
+        let modname = &p.modname()?;
```

Comment:
I don't see what any of the changes to this file accomplish logic wise

Reply:
Somehow when I don't store it as a variable it changes midway and i could not find out why. this was the only was I could make it work